scmailclient/scmail.py

```diff
@@ -0,0 +1,493 @@
+import click
+import pprint
+import re
+from pathlib import Path
+from os import environ
+import getpass
+import logging
+import time
+import requests
+
+
+from gpgopt import GpgOpt
+
+
+# Default to localhost if url is not given
+SECUREMAILBOX_URL = environ.get(
+    "SECUREMAILBOX_URL", "http://securemailbox.duckdns.org:8082/"
```

Comment:
Changing this to `https://securemailbox.duckdns.org/` would close #15.
```suggestion
    "SECUREMAILBOX_URL", "https://securemailbox.duckdns.org"
```

Reply:
Thank you.